pawnShop/src/main/java/com/example/pawnShop/Controller/ProductTypeController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.pawnShop.Controller;
+
+import com.example.pawnShop.Dto.Product.ProductTypeDto;
+import com.example.pawnShop.Service.Contract.ProductTypeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/product_type")
+public class ProductTypeController {
+    private final ProductTypeService productTypeService;
+
+    @GetMapping("/get_all")
+    public ResponseEntity<List<ProductTypeDto>> getAllProductsTypes(){
+        var productsTypes = productTypeService.getAllProductsTypes();
+
+        return ResponseEntity.ok(productsTypes);
+    }
+    @GetMapping("/get_by_id")
```

Comment:
Hi, Toshe @Todorov135 
when requesting an entity record from the DB by Id usually you have to specify it's unique identifier (UUID)
Example:
http://localhost:8080/get_by_id/{UUID} 
Please change the endpoint as this:
@GetMapping(value = "/products/{productid}")